src/blocks/Form/Form.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useCallback, useState} from 'react';
+
+import {InnerForm} from '@gravity-ui/page-constructor';
+
+import {Wrapper} from '../../components/Wrapper/Wrapper';
+import {FormProps} from '../../models/blocks';
+import {PaddingsDirections} from '../../models/paddings';
+import {block} from '../../utils/cn';
+
+import './Form.scss';
+
+const b = block('form-block');
+
+export const Form = ({paddingTop, paddingBottom, ...props}: FormProps) => {
+    const {formData, border = 'shadow'} = props;
+    const [contentLoaded, setContentLoaded] = useState(false);
+
+    const onContentLoad = useCallback(() => {
```

Comment:
I think `useCallback` in this place is not necessary

Reply:
yeap